cmd2/cmd2.py

```diff
@@ -1189,170 +1187,364 @@ def visible_prompt(self) -> str:
+        if final_msg:
+            self.perror(
+                final_msg,
+                end=end,
+                rich_print_kwargs=rich_print_kwargs,
+            )
 
-    def pfeedback(self, msg: Any, *, end: str = '\n') -> None:
-        """Print nonessential feedback.  Can be silenced with `quiet`.
+    def pfeedback(
+        self,
+        *objects: Any,
+        sep: str = " ",
+        end: str = "\n",
+        style: Optional[StyleType] = None,
+        soft_wrap: Optional[bool] = None,
+        rich_print_kwargs: Optional[RichPrintKwargs] = None,
+        **kwargs: Any,  # noqa: ARG002
+    ) -> None:
+        """For printing nonessential feedback. Can be silenced with `quiet`.
 
         Inclusion in redirected output is controlled by `feedback_to_output`.
 
-        :param msg: object to print
-        :param end: string appended after the end of the message, default a newline
+        :param objects: objects to print
+        :param sep: string to write between print data. Defaults to " ".
+        :param end: string to write at end of print data. Defaults to a newline.
+        :param style: optional style to apply to output
+        :param soft_wrap: Enable soft wrap mode. If True, text lines will not be automatically word-wrapped to fit the
+                          terminal width; instead, any text that doesn't fit will run onto the following line(s),
+                          similar to the built-in print() function. Set to False to enable automatic word-wrapping.
+                          If None (the default for this parameter), the output will default to no word-wrapping, as
+                          configured by the Cmd2Console.
+        :param rich_print_kwargs: optional additional keyword arguments to pass to Rich's Console.print().
+        :param kwargs: Arbitrary keyword arguments. This allows subclasses to extend the signature of this
+                       method and still call `super()` without encountering unexpected keyword argument errors.
+                       These arguments are not passed to Rich's Console.print().
         """
         if not self.quiet:
             if self.feedback_to_output:
-                self.poutput(msg, end=end)
+                self.poutput(
+                    *objects,
+                    sep=sep,
+                    end=end,
+                    style=style,
+                    soft_wrap=soft_wrap,
+                    rich_print_kwargs=rich_print_kwargs,
+                )
             else:
-                self.perror(msg, end=end, apply_style=False)
+                self.perror(
+                    *objects,
+                    sep=sep,
+                    end=end,
+                    style=style,
+                    soft_wrap=soft_wrap,
+                    rich_print_kwargs=rich_print_kwargs,
+                )
 
-    def ppaged(self, msg: Any, *, end: str = '\n', chop: bool = False) -> None:
+    def ppaged(
```

Comment:
[Coverage](https://app.codecov.io/gh/python-cmd2/cmd2/pull/1471?src=pr&el=tree&utm_medium=referral&utm_source=github&utm_content=comment&utm_campaign=pr+comments&utm_term=python-cmd2) of `ppaged` seems a tad less than it could be.  Take a look at uncovered lines and see if you could add a unit test.

Reply:
This will inevitably require mocking using something like the `mocker` fixture from `pytest-mock` or `MagicMock` from `unittest`.